common/FileOps.h

```diff
@@ -43,7 +44,7 @@ class FileOps final {
      * Throws FileNotFoundException if path does not exist, and FileNotDirException if path is not a directory.
```

Comment:
maybe mention in this doc comment that if you need to use this single threaded, you can create an empty worker pool?

Reply:
oh nevermind, there appears to be an override for this already.